### PR TITLE
[style] Make inline code padding smaller

### PR DESCRIPTION
**Pull request by zillou**
Old style:

image

New style:

image

### CODE REVIEW

File: _assets/stylesheets/base/_typography.scss [hunk 1]
```diff
@@ -141,6 +141,14 @@
 		font-size: 1.15em;
 		margin: 0 0.25em;
 		padding: 0.25em 0.65em;
+
+    &.highlighter-rouge {
```
Comment (doomspork) [inline _assets/stylesheets/base/_typography.scss]: @zillou could we change this to:

```suggestion
    &.highlighter-rouge {
      padding: 0.2em 0.4em;
      margin: 0;
      font-size: 80%;
    }
```

Comment (doomspork) [reply _assets/stylesheets/base/_typography.scss]: Doesn't look like that worked as intended @zillou, the result is broken code 😞

Comment (zillou) [reply _assets/stylesheets/base/_typography.scss]: Fixed with 8ca0d23

**Comment by doomspork** [comment]
Thank you @zillou 😁

**Pull request merged**